src/Ombi/ClientApp/src/app/requests-list/components/_shared-card-grid.scss

```
@@ -0,0 +1,329 @@
+}
+
+.filter-actions {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.display-count {
+  width: 72px;
+
+  ::ng-deep .mat-form-field-wrapper {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)